### PR TITLE
Refactoring : Create constants

### PR DESCRIPTION
**Pull request by cedricgautier**
- Need to push this  PR https://github.com/KrispyTech/airneis/pull/51 before merging to remove previous commits. (Need to rebase of course)

# Done

- Added constants
- Refactored log code

### CODE REVIEW

File: back/config/db.go [hunk 1]
```diff
@@ -68,24 +68,23 @@ func InitDatabase(config Config, env string) (err error) {
...
 
 	log.Info("Database has been loaded")
 
-	if err := database.AutoMigrate(
+	if err := database.AutoMigrate(&model.Product{},
 		&model.Address{}, &model.Category{}, &model.Contact{},
 		&model.Material{}, &model.Order{}, &model.Product{},
 		&model.Status{}, &model.User{}); err != nil {
-		return errors.Errorf("InitDatabase, unable to auto migrate")
+		return errors.Errorf("initDatabase, unable to auto migrate")
```
Comment (BarnabePILLIAUDIN) [inline back/config/db.go:87]: Shouldn't you use one of the unable you created in the constants you created, or create a new constant

**Review by BarnabePILLIAUDIN — COMMENTED** [review]
Could you please create a small documention about how to use your constants and the heplers you create otherwise we could forget they exist later in the project.

File: back/config/db.go [hunk 2]
```diff
@@ -68,24 +68,23 @@ func InitDatabase(config Config, env string) (err error) {
 
 		database, err = gorm.Open(postgres.Open(databaseURI))
 		if err != nil {
-			return errors.Errorf("InitDatabase, unable to load database %s", err)
+			return errors.Errorf("initDatabase, unable to load database %s", err)
```
Comment (BarnabePILLIAUDIN) [inline back/config/db.go:71]: Shouldn't you use one of the unable you created in the constants you created, or create a new constant

File: back/config/db.go [hunk 3]
```diff
@@ -68,24 +68,23 @@ func InitDatabase(config Config, env string) (err error) {
...
 		}
 
 	default:
 		database, err = InitNeonDatabase(config.Handler.VaultClient)
 		if err != nil {
-			return errors.Wrapf(err, "InitDatabase, unable to load database")
+			return errors.Wrapf(err, "initDatabase, unable to load database")
```
Comment (BarnabePILLIAUDIN) [inline back/config/db.go:77]: Shouldn't you use one of the unable you created in the constants you created, or create a new constant

**Review by hallycsse — APPROVED** [review]
Look good to me.

**Review by Magali-D — APPROVED** [review]
Working well, clear refactoring

File: back/lib/shared/constants/constants.go [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+	Started       string = "started"
+	Unmarshall    string = "unmarshall"
+)
+
+// Phrases
+const (
+	BackConfigBooting string = "Backend configuration booting"
+	BackAppStarted    string = "Backend application, fiber started"
+	HelloWorld        string = "Hello World!"
+	Listen3000        string = ":3000"
```
Comment (cedricgautier) [inline back/lib/shared/constants/constants.go:33]:
```suggestion
```

Comment (cedricgautier) [inline back/lib/shared/constants/constants.go]:
```suggestion
	Port        string = ":3000"
```

File: back/main.go [hunk 5]
```diff
@@ -2,24 +2,25 @@ package main
...
-	app.Get("/", func(c *fiber.Ctx) error {
-		return c.SendString("Hello World!")
+	app.Get(c.HomeRoute, func(ctx *fiber.Ctx) error {
+		return ctx.SendString(c.HelloWorld)
 	})
 
-	log.Info("Routes defined")
-	log.Fatal(app.Listen(":3000"))
+	log.Info(c.RoutesDefined)
+	log.Fatal(app.Listen(c.Listen3000))
```
Comment (cedricgautier) [inline back/main.go]:
```suggestion
	log.Fatal(app.Listen(c.Port))
```

**Review by BarnabePILLIAUDIN — APPROVED** [review]
LGTM!

**Pull request merged**